packages/e2e/e2e_macos/ios/Classes/E2EPlugin.h

```diff
@@ -0,0 +1,4 @@
+#import <Flutter/Flutter.h>
```

Comment:
You shouldn't need iOS classes, just the podspec.

Reply:
Done